taskiq/compat.py

```diff
@@ -13,8 +14,12 @@
 if IS_PYDANTIC2:
     T = TypeVar("T")
 
+    @lru_cache(maxsize=None)
```

Comment:
Don't set it to None, which in long-running task libraries means that the cache will grow indefinitely.

Reply:
I thought the number of parameter types is usually limited. But this is indeed a dangerous action. Should we set it to the default 128, or a large value?

Reply:
I suggest default.